clinica/iotools/converters/genfi_to_bids/genfi_to_bids_utils.py

```diff
@@ -775,16 +791,40 @@ def write_bids(
             metadata["bids_filename"],
             True,
         )
-        if (
-            "dwi" in metadata["bids_filename"]
-            and "Philips" in metadata.manufacturer
-            and dcm2niix_success
-        ):
-            merge_philips_diffusion(
-                to / Path(bids_full_path).with_suffix(".json"),
-                metadata.number_of_parts,
-                metadata.run_num,
+        if dcm2niix_success:
+            scans_filepath = (
+                to
+                / str(metadata.participant_id)
+                / str(metadata.session_id)
+                / f"{metadata.participant_id}_{metadata.session_id}_scan.tsv"
+            )
+            if not Path(scans_filepath).exists():
```

Comment:
`scans_filepath` should already be a `Path` object :

```suggestion
            if not scans_filepath.exists():
```